packages/leap/lib/src/physical_behaviors/collision/collision_detection_behavior.dart

```diff
@@ -48,7 +52,7 @@ class CollisionDetectionBehavior extends PhysicalBehavior {
         world.physicals.where((p) => p.collisionType == CollisionType.standard);
     for (final other in nonMapCollidables) {
       if (intersects(other)) {
-        collisionInfo.otherCollisions.add(other);
+        collisionInfo.otherCollisions?.add(other);
```

Comment:
This behavior isn't optional, is `otherCollisions` is null, we need to set it to `[]` and then `add(other)`

Reply:
```dart
      if (intersects(other)) {
        collisionInfo.otherCollisions ??= [];
        collisionInfo.otherCollisions!.add(other);
      }
```